TESTS/mbed_hal/qspi/flash_configs/S25FL512S_config.h

```diff
@@ -0,0 +1,259 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018-2018 ARM Limited
+ *
```

Comment:
please add SPDX identifiers to new files

Reply:
fixed